test/poolFixtures.hpp

```diff
@@ -456,4 +457,24 @@ TEST_P(umfPoolTest, allocMaxSize) {
     ASSERT_EQ(ptr, nullptr);
 }
 
+TEST_P(umfPoolTest, mallocUsableSize) {
+#ifdef __SANITIZE_ADDRESS__
+    GTEST_SKIP()
+        << "This test is invalid with AddressSanitizer instrumentation";
```

Comment:
Why is this test invalid with ASAN?

Reply:
och, right, @staniewzki, pls add back the comment that was here before